manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -26,7 +26,7 @@ spec:
             summary: The pod disruption budget is preventing further disruption to pods.
             description: The pod disruption budget is at minimum disruptions allowed level. The number of current healthy pods is equal to desired healthy pods.
           expr: |
-            max by(namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_current_healthy == kube_poddisruptionbudget_status_desired_healthy)
+            max by(namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_current_healthy == kube_poddisruptionbudget_status_desired_healthy unless on (namespace, poddisruptionbudget) kube_poddisruptionbudget_labels{label_alerts_openshift_io_pod_disruption_budget_at_limit="disabled"} == 1)
```

Comment:
Why is this needed? Some background would help. We were also thinking of splitting this alert into 2 alerts one for platform and other for customer namespaces.

Reply:
Hi @ravisantoshgudimetla, basically in KubeVirt we have PDBs for VMs that have the number of pods which is equal to the minAvailable parameter at all times, this is by design so that we can live-migrate VMs instead of having k8s evicting them.
 
This alert in that context is just noise that might confuse a cluster-admin and that's why we'd like a way to opt-out from this behavior.

Reply:
@ravisantoshgudimetla i don't think that the split will help in this particular situation since KubeVirt bits are still part of platform and not customer. My only worry is that this might be overused by users to silence their alerts.

Reply:
Why would it be concerning if a user used this label as well?

Isn't it actually better if a user uses this, instead of an admin?

I suspect that there are other alerts who would fire in case of i.e. a drain being blocked

Reply:
> My only worry is that this might be overused by users to silence their alerts.

one option would be to hardcode this specifically for this use case and filter on a PDB name: `poddisruptionbudget=~"kubevirt-disruption-budget-.*"`. But this approach could bring higher maintenance if similar requests come in the future.

Reply:
As we see in the comments, kafka is also impacted by this alert.